src/helm/proxy/clients/huggingface_client.py

```diff
@@ -112,17 +117,25 @@ def __init__(self, cache_config: CacheConfig):
         self.cache = Cache(cache_config)
         self.model_server_instances: Dict[str, HuggingFaceServer] = {}
 
-    def get_model_server_instance(self, model_engine) -> HuggingFaceServer:
-        if model_engine not in self.model_server_instances:
-            if model_engine == "gpt-j-6b":
-                self.model_server_instances[model_engine] = HuggingFaceServer("EleutherAI/gpt-j-6B")
-            elif model_engine == "gpt2":
-                self.model_server_instances[model_engine] = HuggingFaceServer("gpt2")
-            elif model_engine == "santacoder":
-                self.model_server_instances[model_engine] = HuggingFaceServer("bigcode/santacoder")
+    def get_model_server_instance(self, model) -> HuggingFaceServer:
+        if model not in self.model_server_instances:
+            model_config = get_huggingface_model_config(model)
+            if model_config:
+                self.model_server_instances[model] = HuggingFaceServer(model_config)
+            elif model == "EleutherAI/gpt-j-6B":
```

Comment:
can these HF special cases be removed?

Reply:
Eventually, but it's tricky because the HELM model names currently don't match with the Hugging Face ones.

e.g. the model is called `together/gpt-j-6b` in HELM and "EleutherAI/gpt-j-6B" in Hugging Face.